core/src/iam/signin.rs

```diff
@@ -95,114 +97,112 @@ pub async fn signin(
 	}
 }
 
-pub async fn sc(
+pub async fn db(
```

Comment:
Should `super::signin::db` be more descriptive, something like `super::signin::db_rd` or `super::signin::db_record` perhaps?

Reply:
I agree with the naming suggestion. I have changed it to `db_access` in [c6d0d06](https://github.com/surrealdb/surrealdb/pull/3988/commits/c6d0d062abcdda4e9378ec53a21975769163f265). The reason for not referencing records specifically in the name is that signing in with an access method at the database level does not need to be at the record level, it can also be at the database level.